showcase/src/assets/translations/en.json

```diff
@@ -287,6 +287,7 @@
         "WITH_FIXED_HEADER": "Table with fixed header",
         "WITH_TRANSCLUDED_ACTION_BAR": "Table with transcluded Action bar",
         "WITH_CUSTOM_STYLING": "Table with custom styling",
+        "WITH_STICKY_ACTIONS": "Table with sticky row actions",
```

Comment:
Don't forget to change "sticky" by "fixed" in the the translation key and the value